packages/@aws-cdk/aws-kms/README.md

```diff
@@ -21,6 +21,17 @@ new kms.Key(this, 'MyKey', {
 });
 ```
 
+Define a KMS key with waiting period:
```

Comment:
Im not sure that we need to add this to the README of the module, I think the property documentation should be sufficient. I'll leave it to you to decide.